CONTRIBUTING.md

```diff
@@ -10,44 +10,37 @@ Happy coding :dancer:
 ## Requirements :coffee:
 
 - Node version [16 or higher](https://nodejs.org/en/download/) is required
-- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+- A Momento Auth Token is required, you can generate one using the [Momento Console](https://console.gomomento.com)
 
 <br/>
 
-## First-time setup :wrench:
+## Repository Layout
 
-```
-# Install dependencies
-npm install
-```
+This repo contains the source code for two different Momento SDKs: the Node.js SDK, and the Momento Web SDK (for use in browsers). The Node.js SDK uses the `grpc-js` library and communicates with the server via grpc. The Web SDK uses the `grpc-web` library, which requires a server that supports grpc-web. It is capable of communicating via HTTP/1.1 and works in most browsers, where we cannot establish and manage a long-lived TCP connection directly.
 
-<br />
+This repo contains four TypeScript packages, which all reside in the `packages` directory:
 
-## Build :computer:
+- `core`: contains common code that is used by both the Node.js and Web SDKs. The bulk of the code is the definition of the Momento API response types.
+- `common-integration-tests`: contains source code for integration tests that can be run against both the Node.js and Web SDKs.
+- `client-sdk-nodejs`: the source for the Node.js SDK. Has a dependency on `core` and `common-integration-tests`.
+- `client-sdk-web`: the source code for the Web SDK. Has a dependency on `core` and `common-integration-tests`.
 
-```
-npm run build
-```
 
-<br/>
+## Build :computer:
 
-## Linting :flashlight:
+The easiest way to do your initial build of all the packages is to run:
 
 ```
-npm run lint
+./scripts/build-all-packages.sh
 ```
 
-<br/>
-
-## Tests :zap:
+From that point you can change directories into any of the `package` subdirs to work on an individual package. Take a look at the `scripts` section of the `package.json` in each package directory to see what build commands are available. They all support `npm run build` to compile the code. Most also have `npm run unit-test` and `npm run integration-test` to run the unit and integration tests, respectively.
```

Comment:
When running through these commands, I first hit `RESOURCE_EXHAUSTED` on the integration tests. I lowered the concurrency as explained below but it was still a surprise while following the steps here (and I already knew about this). Perhaps we should adjust the script command?

After lowering the concurrency, I the following test failed:
![image](https://github.com/momentohq/client-sdk-javascript/assets/3690240/f719e566-082c-4e0c-92f6-f05191aa89c9)